pkg/networkservice/common/clientinfo/client_test.go

```diff
@@ -0,0 +1,115 @@
+package clientinfo
```

Comment:
Please move tests to `clientinfo_test` package

Reply:
Ok, i will do it